srcts/src/imageutils/createHandlers.ts

```diff
@@ -191,12 +203,12 @@ function createBrushHandler(
     coords.img_css_ratio = coordmap.cssToImgScalingRatio();
 
     // Add variable name mappings
-    coords.mapping = panel.mapping;
+    coords.mapping = panel?.mapping;
 
     // Add scaling information
-    coords.domain = panel.domain;
-    coords.range = panel.range;
-    coords.log = panel.log;
+    coords.domain = panel?.domain;
+    coords.range = panel?.range;
+    coords.log = panel?.log;
```

Comment:
Many usages of `panel?.KEY` as `panel` _could be_ `null`

Reply:
Bang